specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-11-15/imds.json

```diff
@@ -598,6 +606,26 @@
         }
       }
     },
+    "host": {
+      "type": "object",
+      "description": "This contains data about host of VM",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "This is the name of the host of VM. Note that a VM will either have a host or a hostGroup but not both"
+        }
+      }
+    },
+    "hostGroup": {
+      "type": "object",
+      "description": "This contains data about host group of VM",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "This is the name of the host group of VM. Note that a VM will either have a host or a hostGroup but not both"
+        }
+      }
+    },
```

Comment:
I think recommendation is to use camelCase for model name. This change would not affect backend.

Reply:
How's that consistent? The model before it is "SecurityProfile", the one after is "StorageProfile"

Reply:
Sorry was checking something else, corrected